sdks/python/setup.cfg

```diff
@@ -31,24 +31,28 @@ exclude=fast_coders_test|typecoders_test
 [coverage:run]
 branch = True
 source = apache_beam
+omit =
+  # Omit auto-generated files by the protocol buffer compiler.
+  apache_beam/portability/api/*_pb2.py
+  apache_beam/portability/api/*_pb2_grpc.py
 
 [coverage:report]
 # Regexes for lines to exclude from consideration
 exclude_lines =
-    # Have to re-enable the standard pragma
-    pragma: no cover
-    abc.abstractmethod
+  # Have to re-enable the standard pragma
+  pragma: no cover
+  abc.abstractmethod
 
-    # Don't complain about missing debug-only code:
-    def __repr__
-    if self\.debug
+  # Don't complain about missing debug-only code:
+  def __repr__
+  if self\.debug
 
-    # Don't complain if tests don't hit defensive assertion code:
-    raise AssertionError
-    raise NotImplementedError
+  # Don't complain if tests don't hit defensive assertion code:
+  raise AssertionError
```

Comment:
Do we every manually raise this?

Reply:
Thanks for pointing it out. we don't raise it.